source/extensions/filters/network/http_connection_manager/config.cc

```diff
@@ -151,13 +151,18 @@ HttpConnectionManagerConfig::HttpConnectionManagerConfig(
                                                                          context_.listenerScope())),
       proxy_100_continue_(config.proxy_100_continue()),
       delayed_close_timeout_(PROTOBUF_GET_MS_OR_DEFAULT(config, delayed_close_timeout, 1000)),
-      normalize_path_(
-          PROTOBUF_GET_WRAPPED_OR_DEFAULT(config, normalize_path,
-                                          // TODO(htuch): we should have a
-                                          // boolean variant of featureEnabled()
-                                          // here.
-                                          context.runtime().snapshot().featureEnabled(
-                                              "http_connection_manager.normalize_path", 0))) {
+      normalize_path_(PROTOBUF_GET_WRAPPED_OR_DEFAULT(config, normalize_path,
+#ifdef ENVOY_NORMALIZE_PATH_BY_DEFAULT
+                                                      true
```

Comment:
Why not also still allow runtime override? You could just `#ifdef` in the default runtime value?

Reply:
you mean something like 0 to 100? 
The idea is that I am not confident that if runtime is vulnerability. Let me confirm

Reply:
Yes, either 0 or 100 depending on CLI.

Reply:
using the preprocessor to change the default value would work perfectly well for Istio.  thanks